graph-db/src/utils/cmr/fetchPageFromCmr.js

```diff
@@ -0,0 +1,47 @@
+const fetch = require('node-fetch');
+const { getEchoToken } = require('./getEchoToken');
+
+/**
+ * fetchPageFromCMR: Fetch a page of collections from CMR
+ * search endpoint and initiate or continue scroll request
+ * @param scrollId {String} An optional scroll-id given from the CMR
+ * @returns [{JSON}] An array of UMM JSON collection results
+ */
+ exports.fetchPageFromCMR = async scrollId => {
+    const token = null; //getEchoToken();
+    const requestHeaders = {};
+    
+    if (token) {
+      requestHeaders['Echo-Token'] = token;
+    }
+  
+    if (scrollId) {
+      requestHeaders['CMR-Scroll-Id'] = scrollId;
+    }
+    
+    let scrollSession;
+    const response = await fetch(`${process.env.CMR_ROOT}/search/collections.umm_json?page_size=${process.env.PAGE_SIZE}&scroll=true`, {
+      method: 'GET',
+      headers: requestHeaders
+    })
+      .then(response => {
```

Comment:
Should standardize whitespace here, above the `.then` started right after the terminating `)` and this one is offset from others below